ztp/gitops-subscriptions/argocd/example/policygentemplates/group-du-sno-validator-ranGen.yaml

```diff
@@ -20,3 +20,5 @@ spec:
     - fileName: validatorCRs/informDuValidator.yaml
       remediationAction: inform
       policyName: "du-policy"
+      evaluationInterval:
+        compliant: 5s
```

Comment:
As Ian mentioned, can we add some comments here to explain why this short interval value is ok with this policy.